net/interfaces/interfaces_bsd.go

```diff
@@ -40,9 +40,15 @@ func defaultRoute() (d DefaultRouteDetails, err error) {
 // owns the default route. It returns the first IPv4 or IPv6 default route it
 // finds (it does not prefer one or the other).
 func DefaultRouteInterfaceIndex() (int, error) {
+	disabledAlternateDefaultRouteInterface := false
```

Comment:
Plausibly not, but do we want to add an `envknob` as well as a nodeAttr? Specifically for `utun`-mode macOS and/or Headscale?

Reply:
I believe #6566 did not affect those clients -- all OSS versions don't provide an alternate interface function, and we were binding to the default interface for them (see the implementation in `net/netns/netns_darwin_tailscaled.go` from before the change).

Also, I'm hoping these knobs are temporary, so don't necessarily want to add more surface area for them.

Also I was lazy :)